ggml/ggml_convert.py

```diff
@@ -76,13 +81,15 @@ def _nested_getattr(model: Any, name: str) -> Any:
     return node
 
 
-def find_children(model: torch.nn.Module, t: type) -> List[Tuple[str, torch.nn.Module]]:
+def find_children(model: torch.nn.Module, t: type, layer_filter: str = "") -> List[Tuple[str, torch.nn.Module]]:
     queue = list(model._modules.items())
     modules = []
     while queue:
         name, node = queue.pop()
         if node is None:
             continue
+        if layer_filter and not re.match(layer_filter, name):
```

Comment:
Could it filter layer in the ModelList (e.g. something like "layer0 / layer1 /...") ? 'cause this would break the indices of subsequent layer. If not, should we print something to inform the user not to ?

Reply:
it depends how the user regex is constructed. There are probably a lot of way to break the model with this feature, but you can just try again. The PR document a valid regex that specializes the model for speech to text.